microovn/client/client.go

```diff
@@ -148,11 +148,43 @@ func DisableService(ctx context.Context, c *client.Client, serviceName string) e
 func EnableService(ctx context.Context, c *client.Client, serviceName string) error {
 	queryCtx, cancel := context.WithTimeout(ctx, time.Second*30)
 	defer cancel()
-
-	err := c.Query(queryCtx, "PUT", types.APIVersion, api.NewURL().Path("service", serviceName), nil, nil)
+	err := c.Query(queryCtx, "PUT", types.APIVersion, api.NewURL().Path("service","control", serviceName), nil, nil)
 
 	if err != nil {
 		return fmt.Errorf("Failed to enable service '%s': '%s'", serviceName, err)
 	}
 	return nil
 }
+
+// GetServices returns the set of warnings regarding the services
+func GetServiceWarnings(ctx context.Context, c *client.Client) (types.WarningSet, error) {
+	queryCtx, cancel := context.WithTimeout(ctx, time.Second*5)
+	defer cancel()
+
+	warnings := types.WarningSet{}
+
+	err := c.Query(queryCtx, "GET", types.APIVersion, api.NewURL().Path("service","warnings"), nil, &warnings)
+	if err != nil {
+		return warnings, fmt.Errorf("Failed getting warnings: %w", err)
+	}
+
+	return warnings, nil
+}
+
+
+func GetServiceWarningsString(ctx context.Context, c *client.Client) (string, error) {
+	var warnString strings.Builder
+	warnings, err := GetServiceWarnings(ctx,c)
+	if err != nil {
+		return warnString.String(), err
+	}
+
+	if warnings.EvenCentral {
+		warnString.WriteString("Warning: There are an even number of central service nodes, this cannot tolerate any more failures than if you had one less central service node and just leads to decreased write performance\n")
+	}
+
+	if warnings.FewCentral {
+		warnString.WriteString("Warning: There are less than 3 central service nodes, therefore your system cannot tolerate a single node failure putting your system at risk, you should add more central service nodes\n")
```

Comment:
These warnings are rather verbose. Could we make them shorter, focusing on the core issue? Examples:
```
[central] Warning: Cluster with even number of members has same fault tolerance, but higher quorum requirements, than cluster with one less member.
```
```
[central] Warning: Cluster with less than 3 nodes can't tolerate any node failures.
```